_maps/map_files/CTF/cruiser.dmm

```diff
@@ -35,10 +35,6 @@
 /turf/closed/indestructible/syndicate,
 /area/centcom/ctf)
 "cl" = (
-/obj/structure/window/reinforced{
-	dir = 4;
-	resistance_flags = 64
```

Comment:
Hmm need to bring this back, I wonder if you can set these for spawners? If not you should be able to